wordpress-mobile/WordPress-iOS
# Add Me Menu button to My Site Navigation Bar

Ref #13319

Tapping the `Me` button added to the navigation bar, in `My Site` will present the `Me` scene modally
#### Change log
- Add feature flag for `Me Move`
- Add `ScenPresenter` and (concrete implementation) `MeScenePresenter`: responsible of presenting the `Me` scene, injected into `BlogDetailsViewController`
- Add new initializer for `BlogDetailsViewController` (with the above mentioned dependency injection)
- Add extension to `BlogDetailsViewController containing methods to make the new `Me` button
- Since I was working on the `Classes->ViewRelated->Blog` group, I took the occasion to reorganize files and folders in there

#### Notes

1. Known issue: The `Me` image size is not right yet
2. Known issue: when rotating to landscape, the top bars are misaligned in the split view
3. This PR does not remove the original `Me` scene in the tab bar (yet)
4. This PR does not include the gravatar in the `Me` button
5. This PR does not include unit tests

To test:
- Build/run
- Choose a site of your preference and go to `My Site` 
- Check that the `Me` button is visible
- Tap the `Me` button and verify that the `Me` scene is presented modally and works as expected.

PR submission checklist:

  - yet to be added
  - added localized strings where needed, TBD other changes.
  - Too early for that.
